plugins/cache/redis.py

```diff
@@ -73,19 +74,26 @@ def __init__(self, *args, **kwargs):
         try:
```

Comment:
You missed one `connection` a few lines up.

Reply:
Ah! good catch. That'll fix my unit failure too; thanks.